### ISSUE TITLE
 libmblog.so

### ISSUE THREAD

**Issue by Pablito2020** [chenxiaolong/DualBootPatcher#546]
Hello to everybody!
I was compiling a debug build and I had this compilation error:
-- Signing: /home/pablito/DualBootPatcher/build/thirdparty/prebuilts/exfat/1.2.6-1/x86/bin/mount.exfat
/home/pablito/DualBootPatcher/build/hosttools/result/bin/signtool: error while loading shared libraries: libmblog.so: cannot open shared object file: No such file or directory

Of course build/hosttools/result/lib/libmblog.so excists.... So don't know what to do.

PS: Android sdk and ndk patch is successfully added....

Thanks in advance

**Comment by chenxiaolong** [comment]
Sounds like there might be a problem with the rpath. Can you post the output of:

```
ldd /home/pablito/DualBootPatcher/build/hosttools/result/bin/signtool
```

Also, which distro and CMake version are you using?

**Comment by Pablito2020** [comment]
Mmm... The comand says that:
	linux-vdso.so.1 =>  (0x00007fff8b4b0000)
	libmbsign.so => /home/pablito/DualBootPatcher/build/hosttools/result/bin/../lib/libmbsign.so (0x00007f0edf4fb000)
	libcrypto.so.1.0.0 => /lib/x86_64-linux-gnu/libcrypto.so.1.0.0 (0x00007f0edf095000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f0edecce000)
	libmblog.so => not found
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f0edeaca000)
	/lib64/ld-linux-x86-64.so.2 (0x0000558ee3eb7000)

But I can see the file in the ubuntu explorer....
Any idea of that?
Thanks in advance

**Comment by chenxiaolong** [comment]
Thanks for the info! I was able to reproduce the issue in an Ubuntu 17.04 VM. I'll look into it further when I get back from work tomorrow.

**Comment by chenxiaolong** [comment]
@Pablito2020 This has been fixed by #548
